public/app/features/dashboard-scene/scene/layout-default/DashboardGridItem.tsx

```diff
@@ -109,7 +109,7 @@ export class DashboardGridItem
     if (isEqual(this._prevRepeatValues, values)) {
       // In some cases, like for variables that depend on time range, the panel query runners are waiting for the top level variable to complete
       // So even when there was no change in the variable value (like in this case) we need to notify the query runners that the variable has completed it's update
-      this.notifyRepeatedPanelsWaitingForVariables(variable);
+      //    this.notifyRepeatedPanelsWaitingForVariables(variable);
```

Comment:
Shall we just remove this line completely?

Reply:
yes!